### PR TITLE
KOGITO-1918: File doesn't save properly after tried to open an invalid one

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

**Review by ljmotta — APPROVED** [review]
LGTM, bug fixed :rocket:

**Review by tiagobento — APPROVED** [review]

File: packages/desktop/src/__tests__/webview/App.test.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    expect(component.asFragment()).toMatchSnapshot();
+
+    act(() => {
+      electron.ipcRenderer.send("openFile", {
+        file: { filePath: "/a/a.dmn", fileType: "dmn", fileContent: "" }
+      });
+    });
+
+    expect(component.asFragment()).toMatchSnapshot();
+  });
```
Comment (tiagobento) [inline packages/desktop/src/__tests__/webview/App.test.tsx:55]: Actually, I don't think this is correct. AFAIK, you can't have two different snapshots for the same test? Can you double check that please?

**Review by tiagobento — CHANGES_REQUESTED** [review]

File: packages/desktop/src/__tests__/webview/App.test.tsx (hunk 1 above)
Comment (paulovmr) [reply packages/desktop/src/__tests__/webview/App.test.tsx:55]: More than one snapshot works, first one will have the "invalid file type alert alert is closed immediately after leaving the home page 1" identifier and the second will have "invalid file type alert alert is closed immediately after leaving the home page 2".

**Review by tiagobento — APPROVED** [review]

**Comment by domhanak** [comment]
Hi, I wasn't able to reproduce with these changes. The file is properly saved and title is updated after working with invalid file.

**Review by domhanak — APPROVED** [review]

**Review by ederign — APPROVED** [review]

**Pull request merged**